src/Microsoft.OpenApi/Interfaces/IOpenApiReader.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Reader;
+
+namespace Microsoft.OpenApi.Interfaces
+{
+    /// <summary>
+    /// Interface for Open API readers.
+    /// </summary>
+    public interface IOpenApiReader
```

Comment:
Why do we still have this interface?  I thought the idea was to remove the "reader" pattern and replace with Load